### ISSUE TITLE
Reply-To not being populated in email header even though it is built into the form

### ISSUE THREAD

**Issue by Egarbeil** [backdrop-contrib/webform#234]
The Reply-to field is not working - is not being populated in the email headers even though it is in the email form.  When my client clicks on 'Send' it goes to the From address - which is always the site, instead of the email of the person who submitted the form (in the Reply-to field).  I looked and this field is not being populated in the the email header information.  Please advise how to fix this.

Elisabeth

**Comment by argiepiano** [comment]
Hmm... I can't reproduce. I'm getting Reply-To normally. Are you using a mailing contrib module like SMTP perhaps? Some SMTP services do not allow multiple email addresses in the Reply-to field (only one).

<img width="884" alt="Screen Shot 2024-02-26 at 2 19 59 PM" src="https://github.com/backdrop-contrib/webform/assets/9938978/d9ab4db3-35d4-4f01-ae59-4c2adb07261b">

<img width="714" alt="Screen Shot 2024-02-26 at 2 20 24 PM" src="https://github.com/backdrop-contrib/webform/assets/9938978/e2a38e09-9097-451f-a468-a77f02d5a6d7">

**Comment by Egarbeil** [comment]
The Reply-to shows up on the forms, but not in the actual email.  We are using mail system but not mime mail, although it is installed.  I had to turn off due to errors.    Try hitting reply to an email sent by webform and see what email actually comes up. In all cases for us it was the from, not the reply to -  Standard install of Backdrop 1.27x, with mail system, mime mail installed and Webform.  No custom code.  - No reply to in email.  I also tested my email program and if the field is populated it shows up. 

I will provide more details tomorrow evening. 


Be well and stay safe,

Elisabeth Garbeil, MBA
Brainwrap LLC
"Being on the web doesn't have to make your head hurt."
www.brainwrap.com
(810) 560-7181
***@***.***
